### ISSUE TITLE
[Bug] Unable to deselect system

### ISSUE THREAD

**Issue by Bluedagger21** [exodus4d/pathfinder#804]
Connections have the ability to be deselected with `ctrl+click`. Can systems get the same functionality?

**Comment by exodus4d** [comment]
Yes, this is easy to implement, but what is the benefit? I mean, if you deselect a system, modules get removed and we could not use the screen space for other relevant information?

**Comment by Bluedagger21** [comment]
Not sure what the benefit would be. Maybe to keep a Pathfinder tab clean if it's just being used to view a map and not actively use the mapping features? This is very low priority obviously :D
